gsms.py

```diff
@@ -204,6 +232,47 @@ def main() -> None:
                                 'Use the `--apps` arg to specify the full path of the file if you\'d like to use a '
                                 'custom location.')
 
+# Code from here and modified to work in this project
+# https://gist.github.com/mkropat/7550097
+class GUID(ctypes.Structure):
+    _fields_ = [
+        ("Data1", wintypes.DWORD),
+        ("Data2", wintypes.WORD),
+        ("Data3", wintypes.WORD),
+        ("Data4", wintypes.BYTE * 8)
+    ] 
+
+    def __init__(self, uuid_):
+        ctypes.Structure.__init__(self)
+        self.Data1, self.Data2, self.Data3, self.Data4[0], self.Data4[1], rest = uuid_.fields
+        for i in range(2, 8):
+            self.Data4[i] = rest>>(8 - i - 1)*8 & 0xff
+
+class UserHandle:
+    current = wintypes.HANDLE(0)
+    common  = wintypes.HANDLE(-1)
+
+_CoTaskMemFree = windll.ole32.CoTaskMemFree 
```

Comment:
Can you do `ctypes.windll...` to avoid a `from` import?

Same for `wintypes`?

Reply:
`wintypes` is not accessible through `ctypes` itself it seems.
changed it for `windll`